### ISSUE TITLE
Issue with apostrophe (&#8217;) in the doctitle using Asciidoctor.load

### ISSUE THREAD

**Issue by ggenzone** [asciidoctor/asciidoctor#3631]
Hello, 

  I don't know if it is a problem related with some encoding or it is related with the apostrophe symbol.

I can create a simple test to reproduce the issue

```
require 'asciidoctor'

adoc = Asciidoctor.load "= What's happening with you?"
puts adoc.doctitle

```
Expected result:
`What's happening with you?`

Result
`What&#8217;s happening with you?`

```
file test.rb
test.rb: Ruby script, UTF-8 Unicode text
```

```
# lsb_release -a
No LSB modules are available.
Distributor ID: Ubuntu
Description:    Ubuntu 18.04.4 LTS
Release:        18.04
Codename:       bionic
```
Thanks

**Comment by mojavelinux** [comment]
This is expected behavior. It's the built-in smart quotes substitution. You can learn more about replacements here: https://asciidoctor.org/docs/user-manual/#replacements

**Comment by mojavelinux** [comment]
When using the API, you can decode these using the code in this helper from Asciidoctor PDF: https://github.com/asciidoctor/asciidoctor-pdf/blob/master/lib/asciidoctor/pdf/sanitizer.rb#L30-L34

**Comment by ggenzone** [comment]
Perfect I will use that code, and also I will read more about the replacements substitution process.

I use the doctitle to generate a cover page using Prawn, so will decode the doctitle.

Thanks!

**Comment by mojavelinux** [comment]
Yep, we have to do the same think in Asciidoctor PDF when generating the outline. See https://github.com/asciidoctor/asciidoctor-pdf/blob/master/lib/asciidoctor/pdf/converter.rb#L3550-L3552
